.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Switch to file-scoped namespaces
+5f4ff16b4bad507c09bd180091ebbb2df3872279
```

Comment:
How does this work with regard to the fact that PRs get squashed&merged?

Reply:
The commit for this PR, which primarily involves changes in indentation, will be ignored by git-blame. Right?🤔

Reply:
I mentioned about this in a previous comment above, but I shouldn't have added the '.git-blame-ignore-revs' file in this PR. Should I remove it?

Reply:
I think the best course of action here would be to add the `.git-blame-ignore-revs` retroactively

Reply:
@karkarl By "retroactively" you mean remove the file for now and add it later in a new PR?